src/mscorlib/ref/mscorlib.cs

```diff
@@ -2948,6 +2948,7 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         [System.Runtime.InteropServices.ComVisibleAttribute(false)]
         [System.Security.SecuritySafeCriticalAttribute]
         public bool EndsWith(System.String value, System.StringComparison comparisonType) { throw null; }
+        public Boolean EndsWith(String value, Boolean ignoreCase, System.Globalization.CultureInfo culture) { throw null; }
```

Comment:
can we use `bool` primitive type instead, to match the rest? also, probably should use `System.String` fully qualified name.